Assets/Game/Scripts/MainMenu/MainMenu.cs

```diff
@@ -0,0 +1,52 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.UI;
+
+
+namespace GameJammers.GGJ2025.UI {
+    public class MainMenu : MonoBehaviour {
+
+        public int LevelsCleared { get; set; } = 0;
+
+        public string targetScene = "";
+
+        [SerializeField] Button _resumeButton;
+        [SerializeField] GameObject _creditsMenu;
+        [SerializeField] GameObject _startMenu;
+
+
+        void Start () {
+            _resumeButton.gameObject.SetActive(LevelsCleared > 0);
+            _startMenu.SetActive(true);
+            _creditsMenu.SetActive(false);
+        }
+
+        public void LoadGameScene (int level) {
+            var loading = SceneManager.LoadSceneAsync(targetScene, LoadSceneMode.Additive);
```

Comment:
It might be good to do a hard non async load here just in-case somebody does a double click real fast.